### PR TITLE
Low level

### PR DESCRIPTION
**Pull request by Alexy-Def**
<h1>Task 3. Low level. Currency converter</h1>

This program should convert one currency to another. The program has 4 currencies.

- [x] Currency
- [x] Simple calculator
- [x] Hard calculator (with splitting string)

### CODE REVIEW

File: TMS.Net07.Homework3.CurrencyAndCalculator/TMS.Net07.Homework3.Low.Currency/TMS.Net07.Homework3.CurrencyAndCalculator/Program.cs [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+                string currencyFrom = Console.ReadLine().ToLower();
+
+                Console.Write("Input currency TO: ");
+                string currencyTo = Console.ReadLine().ToLower();
+
+                Console.Write("Choice BUY/SELL: ");
+                string buyOrSell = Console.ReadLine().ToLower();
+
+                Console.Write("Input amount: ");
+                bool isParseAmount = float.TryParse(Console.ReadLine(), out float currencyAmount);
```
Comment (radzevich) [inline TMS.Net07.Homework3.CurrencyAndCalculator/TMS.Net07.Homework3.Low.Currency/TMS.Net07.Homework3.CurrencyAndCalculator/Program.cs]: For money it's better to use `decimal` for higher accuracy.
Initially, `decimal` was invented for money

File: TMS.Net07.Homework3.CurrencyAndCalculator/TMS.Net07.Homework3.Low.Currency/TMS.Net07.Homework3.CurrencyAndCalculator/Program.cs [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+                if (incorrectlyInputBuyOrSell != " ")
+                {
+                    Console.WriteLine();
+                    Console.WriteLine(incorrectlyInputBuyOrSell);
+                }
+
+                Console.WriteLine();
+                Console.Write("Exit or Not: ");
+                string exit = Console.ReadLine().ToLower();
+                if (exit == "exit") return;
```
Comment (radzevich) [inline TMS.Net07.Homework3.CurrencyAndCalculator/TMS.Net07.Homework3.Low.Currency/TMS.Net07.Homework3.CurrencyAndCalculator/Program.cs]:
```suggestion
                if (exit == "exit") { return; }
```

**Review by radzevich — CHANGES_REQUESTED** [review]
Good job! Just a minor improvements required

File: TMS.Net07.Homework3.CurrencyAndCalculator/TMS.Net07.Homework3.Low.Currency/TMS.Net07.Homework3.CurrencyAndCalculator/Program.cs (hunk 1 above)
Comment (Alexy-Def) [reply TMS.Net07.Homework3.CurrencyAndCalculator/TMS.Net07.Homework3.Low.Currency/TMS.Net07.Homework3.CurrencyAndCalculator/Program.cs]: Ok, I changed.

File: TMS.Net07.Homework3.CurrencyAndCalculator/TMS.Net07.Homework3.Low.Currency/TMS.Net07.Homework3.CurrencyAndCalculator/Program.cs (hunk 2 above)
Comment (Alexy-Def) [reply TMS.Net07.Homework3.CurrencyAndCalculator/TMS.Net07.Homework3.Low.Currency/TMS.Net07.Homework3.CurrencyAndCalculator/Program.cs]: I completely forgot about brackets :(
I changed.

**Review by radzevich — APPROVED** [review]

**Pull request merged**